service-app/internal/auth/middleware_test.go

```diff
@@ -0,0 +1,79 @@
+package auth
+
+import (
+	"context"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/ministryofjustice/opg-scanning/config"
+	"github.com/ministryofjustice/opg-scanning/internal/logger"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAuthenticateMiddleware(t *testing.T) {
+	// @TODO implement test case checking token/cookie validity
```

Comment:
Does this still need to be done?

Reply:
Yes, its work that needs to be done to improve test coverage, but im not planning to block this PR for it.

Reply:
@gregtyler ended up removed this todo in favour of segregated testing for authenticator: https://github.com/ministryofjustice/opg-scanning/pull/29/commits/c0745bc937a6b041054f4c1c0ce212ec336aa995